Lang/Spanish.pj.Lang

```diff
@@ -177,7 +178,7 @@
 #320#  "Seleccionar directorio de ROMs actual"
 
 //Messages
-#340#  "¿ROM Dañado? Usa GoodN64 y verifica que la RDB esté actualizada."
+#340#  "¿ROM Dañada? Usa GoodN64 y comprueba por una RDB actualizada."

```

Comment:
Let me ask you this.
What part of the context is lost if you use 'comprueba que la RDB esté actualizada'?

Is the -idea- communicated in Spanish different from the idea that would be communicated in English?